src/radio/index.tsx

```diff
@@ -15,7 +16,7 @@ export interface RadioProperties extends ThemeProperties, FocusProperties {
 	/** Set the disabled property of the control */
 	disabled?: boolean;
 	/** Adds a <label> element with the supplied text */
-	label?: string;
+	label?: DNode;
```

Comment:
If anything, this should be `RenderResult`, we need to re-visit what we're going to do for these kind of labels